src/app/app.ts

```diff
@@ -0,0 +1,28 @@
+import type { BaseComponent } from '@components/base-component';
+
+import { PageWrapper } from './page';
+
+class App {
+  constructor(
+    private pageWrapper: BaseComponent,
+    private root: HTMLElement,
+  ) {}
+
+  public stop(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+
+  public start(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+
+  public pause(): void {
+    this.root.append(this.pageWrapper.getNode());
+  }
+}
+const app = new App(PageWrapper(), document.querySelector<HTMLDivElement>('#app')!);
```

Comment:
Using ! with document.querySelector can lead to runtime errors if #app is not found. Consider adding a check before passing it to App
```suggestion
const rootElement = document.querySelector<HTMLDivElement>('#app');
if (!rootElement) {
  throw new Error('Root element #app not found');
}
const app = new App(PageWrapper(), rootElement);
```